src/forms/renderers/MissingType.tsx

```diff
@@ -31,7 +31,7 @@ const MissingTypeRenderer = (props: LayoutProps) => {
     return (
         <Hidden xsUp={!visible}>
             <Alert severity="error">
-                Error: Invalid field definition at
+                <span>Error: Invalid field definition at</span>
```

Comment:
An eslint rule to handle the translate issues called this out and it seemed fair.